src/environment.ts

```diff
@@ -92,6 +95,19 @@ export class SecureEnvironment implements MembraneBroker {
             this.createSecureRecord(secure, raw);
             this.createSecureRecord(secure.prototype, raw.prototype);
         }
+        // remapping intrinsics that are not reachable via global names
+        const [
+            SecureAsyncFunction,
+            SecureGeneratorFunction,
+            SecureAsyncGeneratorFunction,
+        ] = secureGlobalThis.eval(`[
```

Comment:
I was trying to figure a way to share this... for both environments... maybe we can use eval for the outer realm, and then use the same mechanism to extra them both.